app/assets/stylesheets/modules/projects.css.scss

```diff
@@ -426,6 +426,20 @@ $card-background: rgba(235,235,235,1);
     .modal-body {
       background: #f2f2f2;
       padding: 30px;
+      padding-bottom: 20px;
+    }
+
+    p.content {
```

Comment:
@trevorlang, I have some thoughts on getting work like this done. We can discuss during our next retrospective. The TL;DR is that given our team setup/resources, and given what Ryan does best, we can't afford him digging into CSS optimization. But, that _is_ what you do best. So moving forward I want to work out some sort of tag-team approach to ensuring top-notch CSS.

For the PR, we'll let it slide. But in the next E2E services PR we will start dedicating a little time for you to ensure things are optimized for the project page/E2E branch. Again, we can discuss specifics next week. It might be a huge disaster, but it's worth a shot. I think there's a lot to be gained doing it this way.